client/src/app/pages/tasks/useTaskActions.tsx

```diff
@@ -69,15 +69,27 @@ export const useTaskActions = (task: Task) => {
   return [
     {
       title: t("actions.cancel"),
-      isDisabled: !canCancel(task.state),
+      isAriaDisabled: !canCancel(task.state),
+      tooltipProps: {
+        content: !canCancel(task.state)
+          ? `${t("message.cancelNotAvailable", { statusName: task.state })}`
```

Comment:
note that `t()` function already returns a string -  no need to use template literals.